src/telliot_feeds/sources/manual_sources/spot_price_input_source.py

```diff
@@ -0,0 +1,54 @@
+from dataclasses import dataclass
+
+from telliot_feeds.datasource import DataSource
+from telliot_feeds.dtypes.datapoint import DataPoint
+from telliot_feeds.dtypes.datapoint import datetime_now_utc
+from telliot_feeds.utils.log import get_logger
+
+
+logger = get_logger(__name__)
+
+
+@dataclass
+class SpotPriceManualSource(DataSource[float]):
+    """DataSource for a spot-price manually-entered data."""
+
+    def parse_user_val(self) -> float:
+        """Parse user input for a spot price value."""
+
+        print("\nType your spot price and press [ENTER].\n\nFor example, if price is $1234.0, type 1234 or 1234.0\n")
+
+        spot = None
+
+        while spot is None:
+            usr_inpt = input("$ ")
+
+            try:
+                usr_inpt = float(usr_inpt)  # type: ignore
+            except ValueError:
+                print("Invalid input. Enter decimal value (float).")
+                continue
+
+            print(f"\nSpot price (with 18 decimals of precision) to be submitted on chain: {int(usr_inpt*10**18)}")
```

Comment:
Ensure that it doesn't print scientific notation to the console. Might need to add `:f` before the closing curly bracket in your f-string.